frontend/src/lib/components/TestJobLoader.svelte

```diff
@@ -172,13 +186,38 @@
 			try {
 				if (job && `running` in job) {
 					const offset = logOffset == 0 ? (job.logs?.length ? job.logs?.length + 1 : 0) : logOffset
+
+					var getProgress: boolean | undefined = undefined;
+
+					// We only pull individual job progress this way
+					// Flow's progress we are getting from FlowStatusModule of flow job
+					if (job.job_kind == "script" || "preview")
+						if (lastTimeCheckedProgress){
+							// Ask for progress if the last time we asked is >5s OR the progress was once not undefined
+							if ((Date.now() - lastTimeCheckedProgress > getProgressDelay) || scriptProgress != undefined)
+								lastTimeCheckedProgress = Date.now(),	getProgress = true;																	
+						} else 
```

Comment:
always use { }, even for one-liners, otherwise error-prone

Reply:
Fixed